library/Zend/Log/Writer/Mail.php

```diff
@@ -86,6 +86,9 @@ public function __construct($mail, Transport\TransportInterface $transport = nul
             }
             $transport = isset($mail['transport']) ? $mail['transport'] : null;
             $mail      = isset($mail['mail']) ? $mail['mail'] : null;
+            if (is_array($mail)) {
```

Comment:
We need this verification.
First : check that $mail is an array.
Then : $mail = $mail['mail']

So this line checks that $mail['mail'] is an array.